tests/test_tiatoolbox.py

```diff
@@ -1,37 +1,115 @@
 #!/usr/bin/env python
 
 """Tests for `tiatoolbox` package."""
-
 import pytest
 
-from click.testing import CliRunner
-
-from tiatoolbox import tiatoolbox
+from tiatoolbox.dataloader.slide_info import slide_info
+from tiatoolbox.utils import misc_utils as misc
 from tiatoolbox import cli
+from tiatoolbox import __version__
+
+from click.testing import CliRunner
+import requests
+import os
+import pathlib
 
 
 @pytest.fixture
-def response():
-    """Sample pytest fixture.
+def response_ndpi(request):
+    """
+    Sample pytest fixture for ndpi images
+    Download ndpi image for pytest
+    """
+    ndpi_file_path = pathlib.Path(__file__).parent.joinpath("CMU-1.ndpi")
+    if not pathlib.Path.is_file(ndpi_file_path):
+        r = requests.get(
+            "http://openslide.cs.cmu.edu/download/openslide-testdata/Hamamatsu/CMU-1.ndpi"
+        )
+        with open(ndpi_file_path, "wb") as f:
+            f.write(r.content)
+
+    def close_ndpi():
+        if pathlib.Path.is_file(ndpi_file_path):
+            os.remove(str(ndpi_file_path))
 
-    See more at: http://doc.pytest.org/en/latest/fixture.html
+    request.addfinalizer(close_ndpi)
+    return response_ndpi
+
+
+@pytest.fixture
+def response_svs(request):
+    """
+    Sample pytest fixture for svs images
+    Download ndpi image for pytest
     """
-    # import requests
-    # return requests.get('https://github.com/audreyr/cookiecutter-pypackage')
+    svs_file_path = pathlib.Path(__file__).parent.joinpath("CMU-1.svs")
+    if not pathlib.Path.is_file(svs_file_path):
+        r = requests.get(
+            "http://openslide.cs.cmu.edu/download/openslide-testdata/Hamamatsu/CMU-1.ndpi"
+        )
+        with open(svs_file_path, "wb") as f:
+            f.write(r.content)
 
+    def close_ndpi():
+        if pathlib.Path.is_file(svs_file_path):
+            os.remove(str(svs_file_path))
 
-def test_content(response):
-    """Sample pytest test function with the pytest fixture as an argument."""
+    request.addfinalizer(close_ndpi)
+    return response_svs
+
+
+def test_slide_info(response_ndpi, response_svs):
+    """
+    pytest for slide_info as a python function
+    """
     # from bs4 import BeautifulSoup
     # assert 'GitHub' in BeautifulSoup(response.content).title.string
```

Comment:
Should this be in here?

Reply:
Removed in this commit https://github.com/TIA-Lab/tiatoolbox/commit/b9b6dd5f9940591a52441381911555f9348ae566